### PR TITLE
modify finetune tutorial

### PR DESCRIPTION
**Pull request by yuedongli1**
Thank you for your contribution to the MindYOLO repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindyolo/blob/master/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: examples/finetune_SHWD/README.md [hunk 1]
```diff
@@ -41,20 +43,18 @@
   ```shell
   python examples/finetune_SHWD/convert_shwd2yolo.py --root_dir /path_to_shwd/SHWD
```
Comment (CaitinZhao) [inline examples/finetune_SHWD/README.md:44]: 放下你的数据集长啥样，怎么转换的，关键代码直接贴上去

File: examples/finetune_SHWD/finetune_shwd.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def train_shwd(args):
         ema = EMA(network, ema_network)
     else:
         ema = None
-    load_pretrain(network, args.weight, ema, args.ema_weight)  # load pretrain
+    load_pretrain(network, args.weight, ema, args.ema_weight, strict=False)  # load pretrain
```
Comment (CaitinZhao) [inline examples/finetune_SHWD/finetune_shwd.py:51]: 这个文件能不要吗，直接复用mindyolo的train的脚本

File: examples/finetune_SHWD/README.md [hunk 3]
```diff
@@ -41,20 +43,18 @@
...
-
-由于SHWD数据集只有7000+张图片，选择yolov7-tiny进行该数据集的训练，可下载MindYOLO提供的在coco数据集上训练好的[模型文件](https://github.com/mindspore-lab/mindyolo/blob/master/MODEL_ZOO.md)作为预训练模型。由于coco数据集含有80种物体类别，SHWD数据集只有两类，模型的最后一层head层输出与类别数nc有关，因此需将预训练模型文件的最后一层去掉， 可参考[convert_yolov7-tiny_pretrain_ckpt.py](./convert_yolov7-tiny_pretrain_ckpt.py)。运行方式如下：
+#### 编写yaml配置文件
+配置文件主要包含原数据地址、数据增强、loss、optimizer、模型结构涉及的相应参数，可参考[yolov7-tiny_shwd.yaml](./yolov7-tiny_shwd.yaml)。
 
-  ```shell
-  python examples/finetune_SHWD/convert_yolov7-tiny_pretrain_ckpt.py
-  ```
+#### 下载预训练模型(建议)
+可选用MindYOLO提供的[MODEL_ZOO](../../MODEL_ZOO.md)作为自定义数据集的预训练模型，预训练模型在COCO数据集上已经有较好的精度表现，相比从头训练，加载预训练模型一般会拥有更快的收敛速度以及更高的最终精度，并且大概率能避免初始化不当导致的梯度消失、梯度爆炸等问题。
```
Comment (CaitinZhao) [inline examples/finetune_SHWD/README.md:52]: 这个不是建议，finetune的教程这步是必须的

File: examples/finetune_SHWD/README.md [hunk 4]
```diff
@@ -20,52 +20,102 @@
...
 
-#### 预训练模型文件转换
+#### 编写yaml配置文件
+配置文件主要包含数据集、数据增强、loss、optimizer、模型结构涉及的相应参数，由于MindYOLO提供yaml文件继承机制，可只将需要调整的参数编写为yolov7-tiny_shwd.yaml，并继承MindYOLO提供的原生yaml文件即可，其内容如下：
+```
+__BASE__: [
+  '../../configs/yolov7/yolov7-tiny.yaml',
+]
+
+per_batch_size: 16 # 16 * 8 = 128
```
Comment (CaitinZhao) [inline examples/finetune_SHWD/README.md:75]: # 单卡batchsize，总的batchsize=per_batch_size * device_num

File: examples/finetune_SHWD/README.md [hunk 5]
```diff
@@ -20,52 +20,102 @@
...
+img_size: 640 # image sizes
+weight: ./yolov7-tiny_pretrain.ckpt
+strict_load: False
```
Comment (CaitinZhao) [inline examples/finetune_SHWD/README.md:78]: strict_load: False  # 是否按严格加载ckpt内参数，默认True，若设成False，当分类数不一致，丢掉最后一层分类器的weight

File: examples/finetune_SHWD/README.md [hunk 6]
```diff
@@ -20,52 +20,102 @@
...
+log_interval: 10
```
Comment (CaitinZhao) [inline examples/finetune_SHWD/README.md:79]: log_interval: 10  # 每log_interval次迭代打印一次loss结果

File: examples/finetune_SHWD/README.md [hunk 7]
```diff
@@ -20,52 +20,102 @@
...
+
+data:
+  dataset_name: shwd
+  train_set: ./SHWD/train.txt
```
Comment (CaitinZhao) [inline examples/finetune_SHWD/README.md:83]: train_set: ./SHWD/train.txt  # 实际训练数据路径

File: examples/finetune_SHWD/README.md [hunk 8]
```diff
@@ -20,52 +20,102 @@
...
+  val_set: ./SHWD/val.txt
+  test_set: ./SHWD/val.txt
+  nc: 2
```
Comment (CaitinZhao) [inline examples/finetune_SHWD/README.md:86]: nc: 2  # 分类数

File: examples/finetune_SHWD/README.md [hunk 9]
```diff
@@ -20,52 +20,102 @@
...
+  # class names
+  names: [ 'person',  'hat' ]
```
Comment (CaitinZhao) [inline examples/finetune_SHWD/README.md:88]: names: [ 'person',  'hat' ]  # 每一类的名字

**Review by CaitinZhao — APPROVED** [review]

**Review by zhanghuiyao — APPROVED** [review]
